src/Neo/SmartContract/ContractBasicMethod.cs

```diff
@@ -0,0 +1,122 @@
+        /// }
+        /// </code>
+        /// </summary>
+        public static string Verify { get; } = "verify";
+
+        /// <summary>
+        /// The initialization method. Compiled into the <see cref="Manifest"/> file if any function uses the initialize statement.
+        /// These functions are executed first when loading the contract.
+        /// Example:
+        /// <code>
+        ///     private static readonly UInt160 owner = "NdUL5oDPD159KeFpD5A9zw5xNF1xLX6nLT";
+        /// </code>
+        /// </summary>
+        public static string Initialize { get; } = "_initialize";
+
+        /// <summary>
+        /// The deployment method. Automatically executed by the ContractManagement contract when a contract is first deployed or updated.
+        /// <code>
+        /// {
+        ///     "name": "_deploy",
+        ///     "safe": false,
+        ///     "parameters": [
+        ///     {
+        ///         "name": "data",
+        ///         "type": "Any"
+        ///     },
+        ///     {
+        ///         "name": "update",
+        ///         "type": "Boolean"
+        ///     }
+        ///     ],
+        ///     "returntype": "Void"
+        /// }
+        /// </code>
+        /// </summary>
+        public static string Deploy { get; } = "_deploy";
+
+        /// <summary>
+        /// The update method. Requires <see cref="NefFile"/> or <see cref="Manifest"/>, or both, and is passed to _deploy.
+        /// Should verify the signer's address using SYSCALL <code>Neo.Runtime.CheckWitness</code>.
+        /// <code>
+        /// {
+        ///   "name": "update",
+        ///   "safe": false,
+        ///   "parameters": [
+        ///     {
+        ///       "name": "nefFile",
+        ///       "type": "ByteArray"
+        ///     },
+        ///     {
+        ///       "name": "manifest",
+        ///       "type": "ByteArray"
+        ///     },
+        ///     {
+        ///       "name": "data",
+        ///       "type": "Any"
+        ///     }
+        ///   ],
+        ///   "returntype": "Void"
+        /// }
+        /// </code>
+        /// </summary>
+        public static string Update { get; } = "update";
+
+        /// <summary>
+        /// The destruction method. Deletes all the storage of the contract.
+        /// Should verify the signer's address using SYSCALL <code>Neo.Runtime.CheckWitness</code>.
+        /// Any tokens in the contract must be transferred before destruction.
+        /// <code>
+        /// {
+        ///   "name": "destroy",
+        ///   "safe": false,
+        ///   "parameters": [],
+        ///   "returntype": "Void"
+        /// }
+        /// </code>
+        /// </summary>
+        public static string Destroy { get; } = "destroy";
+
+        /// <summary>
+        /// Parameter counts for the methods.
+        /// -1 represents the method can take arbitrary parameters.
+        /// </summary>
+        public static int VerifyPCount { get; } = -1;
```

Comment:
Any ?

Reply:
Currently its the only method that takes any number of parameters, so i would say its not necessary.